tests/test_core/test_new_alignment.py

```diff
@@ -1538,6 +1957,18 @@ def test_sequence_collection_strand_symmetry():
     assert numpy.allclose(result["seq2"].observed.array, [[3, 0], [2, 1]])
 
 
+@pytest.mark.xfail(reason="todo: kath, decide on how to handle rename_seqs")
+def test_sequence_collection_rename_seqs():
```

Comment:
i think rename has to involve a new SeqsData instance. Since the underlying data is being made immutable, the only memory overhead are new names and new dictionaries.

Reply:
will do in next PR